src/EventListener/RequestConfigurationSubscriber.php

```diff
@@ -91,4 +91,15 @@ public function onKernelController(ControllerEvent $event): void
         $request->setRequestFormat('json');
         $request->attributes->set(self::ATTRIBUTE_API, true);
     }
+
+	private function getAnnotation(\ReflectionMethod $reflection, string $annotationClass)
```

Comment:
It looks like this function can be pretty annotated with psalm generics annotations, you receive a class-string and return an instance of this class-string